libs/data-mapper/src/lib/utils/__test__/ConnectionUtils.spec.ts

```diff
@@ -89,7 +88,7 @@ describe('utils/Connections', () => {
     });
   });
 
-  describe('addNodeToConnections (primarily drawn connections)', () => {
+  describe('setConnectionInputValue - Drawn connections', () => {
```

Comment:
Nit: make this describe just "setConnectionInputValue" and then layer in another describe "Drawn Connections" and then below do the same for input drop down